compareBuckets/compareBuckets.js

```diff
@@ -224,15 +220,14 @@ function compareBuckets(params, log, cb) {
                 return process.nextTick(() => done(null));
             });
         },
-        () => (!srcDone || !dstDone || srcContents.length > 0 || dstContents.length > 0),
+        async () => (!srcDone || !dstDone || srcContents.length > 0 || dstContents.length > 0),
         err => {
             statusObj.dstBucketInProgress = null;
             statusObj.srcBucketInProgress = null;
-            cb(err);
+            cb(err, statusObj);
```

Comment:
callback "result" is not used (only the first param, err), so why do we need to add it?
(this variable is actually "shared" which the caller, who provides it when calling the function)